src/api/v1/minecraft.go

```diff
@@ -0,0 +1,188 @@
+package v1
+
+import (
+	"crypto/sha256"
+	"encoding/hex"
+	"errors"
+	"fmt"
+	"github.com/labstack/echo"
+	"io/ioutil"
+	"net/http"
+	"sort"
+	"strings"
+)
+
+type (
```

Comment:
just because you can doesn't mean you should 😹 

(i dont mind much at all) but this type struct is a big meme

Reply:
I personally find grouping multiple type declarations together more readable /shrug

Reply:
ye its fine just a little unorthodox =)